### PR TITLE
feat: Parameterise 37 coords ref file

### PR DESCRIPTION
**Pull request by tommyli**
This relates to Seqr [issue #1010](https://github.com/broadinstitute/seqr/issues/1010).  It's also an extension to #204 by parameterising the liftover file path.

(cherry picked from commit 2a8a4ebb21f42fe68f12c4ddd20648567d231018)

### CODE REVIEW

**Review by hanars — APPROVED** [review]
Waiting for Mike's approval before merging

File: luigi_pipeline/lib/hail_tasks.py
Comment (mike-w-wilson) [inline luigi_pipeline/lib/hail_tasks.py:186]:
```suggestion
    def add_37_coordinates(mt, liftover_ref_path):
        """Annotates the GRCh38 MT with 37 coordinates using hail's built-in liftover
        :param mt: MatrixTable from VCF
        :param liftover_ref_path: Path to GRCh38 to GRCh37 coordinates file
        :return: MatrixTable annotated with GRCh37 coordinates
```

**Review by mike-w-wilson — APPROVED** [review]
Small suggestion for the docstring but this looks good to me.

**Pull request merged**
